load_screen_fails_and_caregivers_into_screen_follow_up.R

```diff
@@ -63,6 +63,14 @@ rc_upload <- screen_fails_and_caregivers %>%
 
 # specify local context for data upload
 #load_dot_env('local.env')
-redcap_write(rc_upload,
+result <- redcap_write(rc_upload,
         redcap_uri = Sys.getenv("URI"),
         token = Sys.getenv("SCREEN_TOKEN"))
+
+message <- paste0("Uploaded rows: ", nrow(rc_upload),
+        "\nAffected records: ", result$records_affected_count
+        )
```

Comment:
```suggestion
message <- paste0(nrow(rc_upload), " screen fail records in input file\n",
        result$records_affected_count, " records written to ", Sys.getenv("SCREEN_PROJECT_TITLE")
        )
```

The message content I suggested was robotic. I think this is better suited for the puny humans

Reply:
🤖 `THAT MESSAGE CONTENT WAS CREATED ORGANICALLY BY MY FAT-BASED ELECTROCHEMICAL COMPUTING CENTER. OFFENSE TAKEN. RELATION PENALTY 3.000000000000000001`

Reply:
Really, I thought it took my level of expertise to write something like that.